src/components/AdminSettings.vue

```diff
@@ -339,6 +561,35 @@ export default {
 					this.formMode.ncOauth = F_MODES.VIEW
 					this.isFormCompleted.ncOauth = true
 				}
+				if (this.formMode.ncOauth === F_MODES.VIEW) {
+					this.showDefaultManagedProjectFolders = true
+				}
+				if (this.showDefaultManagedProjectFolders) {
+					this.formMode.projectFolderSetUp = F_MODES.VIEW
+					this.isFormCompleted.projectFolderSetUp = true
+				}
+				if (this.state.app_password_set) {
+					this.formMode.opUserAppPassword = F_MODES.VIEW
+					this.isFormCompleted.opUserAppPassword = true
+					this.currentProjectFolderState = true
+					this.isProjectFolderSwitchEnabled = true
+					this.textLabelProjectFolderSetupButton = this.buttonTextLabel.keepCurrentChange
+				}
+				this.isProjectFolderSwitchEnabled = this.currentProjectFolderState === true
+			}
+		},
+		projectFolderSetUpErrorMessageDescription(errorKey) {
+			switch (errorKey) {
+			case 'The group folder name "OpenProject" integration already exists' :
```

Comment:
Recent changes in figma. Can be done in coming PR's. 
```suggestion
			case 'The group folder name "OpenProject" already exists' :
```